lobby/src/server.rs

```diff
@@ -1,37 +1,41 @@
-#![feature(proc_macro_hygiene, decl_macro)]
+#![feature(decl_macro)]
+#![feature(proc_macro_hygiene)]
 
 use std::vec::Vec;
 
-#[macro_use] extern crate rocket;
-#[macro_use] extern crate serde_derive;
 use rocket::fairing::AdHoc;
 use rocket::http::Header;
 use rocket_contrib::json::Json;
 
-use serde::Serialize;
+use serde_derive::Serialize;
+
+use rocket::{get, routes};
 
 // this is just here for a POC.
 // TODO move those into their own file
 #[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
 struct GameType {
     name: String,
     icon: String,
-    displayName: String,
+    display_name: String,
 }
 
 #[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
 struct Game {
     name: String,
     type_: String,
```

Comment:
I suggest using 
```rust
#[serde(rename = "type")]
type_:String,
```
to avoid needing to rename the variable on the frontend

Reply:
Ahhh, nice, this is still from my very first attempt where I didn't know about all the fancy serde helpers :sweat_smile: